sell/sell-api.yaml

```
@@ -185,11 +195,17 @@ paths:
           description: The ID of the deal the activity is linked to.
           schema:
             type: string
-        - name: createdAt
+        - name: associatedUserId
```

```suggestion
        - name: associatedUserIds
```
To match with the current Activity entity

---

Probably here should be singular because we are looking for the one associate user id, no?